### PR TITLE
Seneca auth

### PR DESCRIPTION
**Pull request by ckiss**
@mcdonnelldean here is the seneca-auth stuff in place and I took the liberty to add lab and linting tools + fixed the errors :) Hope this is what you expected. Let me know if you have any concerns

### CODE REVIEW

**Comment by mcdonnelldean** [comment]
Epic stuff, can you add a travis file too? So I need to pull concorda for this to work right?

**Comment by ckiss** [comment]
@mcdonnelldean no, not yet. This just uses seneca-auth to handle the login and logout. I'll add the travis file so don't merge this just yet :)

**Comment by mcdonnelldean** [comment]
@ckiss Perfect

**Comment by ckiss** [comment]
@mcdonnelldean travis file is added now. can you please review and merge?

**Comment by mcdonnelldean** [comment]
Looks good but let's hold off until @mirceaalexandru makes his agreed changes based on the mail today

File: server/index.js [hunk 1]
```diff
@@ -35,6 +46,18 @@ var plugins = [
 // Register our plugins, kick off the server
 // if there is no error.
 server.register(plugins, function (err) {
+  var seneca = server.seneca
+
+  seneca.use(Auth, {
+    restrict: '/api',
+    server: 'hapi',
```
Comment (mcdonnelldean) [inline server/index.js]: Gone

File: server/index.js [hunk 2]
```diff
@@ -35,6 +46,18 @@ var plugins = [
...
+    strategies: [
+      {
+        provider: 'local'
+      }
```
Comment (mcdonnelldean) [inline server/index.js]: Gone

**Pull request merged**